fabtests/common/shared.c

```diff
@@ -504,6 +505,19 @@ void ft_free_host_tx_buf(void)
 	tx_msg_buf = NULL;
 }
 
+size_t ft_get_aligned_size(size_t size, size_t alignment)
+{
+	return ((size % alignment) == 0) ?
+		size : ((size / alignment) + 1) * alignment;
+}
+
+void *ft_get_aligned_addr(void *ptr, size_t alignment)
+{
+	/* alignment must be power of 2, hence the assertion */
+	assert((alignment & (alignment - 1)) == 0);
+	return (void *)(((uintptr_t)ptr + alignment - 1) & ~((uintptr_t)alignment - 1));
+}
```

Comment:
The above functions can be static, but I'd actually move them into the header file as static inline, to save the trouble of doing that further down the road.